src/cinder/Utilities.cpp

```diff
@@ -76,20 +76,22 @@ std::vector<std::string> split(std::string str, const std::string &separators, b
 	
 	
 	std::size_t searchPrevPos = 0, searchPos;
+	
 	while ((searchPos = str.find_first_of(separators, searchPrevPos)) != std::string::npos)
 	{
-		if (searchPos >= searchPrevPos && !compress)
-			result.push_back(str.substr(searchPrevPos, searchPos - searchPrevPos + 1));
+		
+		if (searchPos >= searchPrevPos && !compress){
+			result.push_back(str.substr(searchPrevPos, searchPos - searchPrevPos));
+		}
 		else if(searchPos > searchPrevPos){
 			result.push_back(str.substr(searchPrevPos, searchPos - searchPrevPos));
 		}
 		
 		searchPrevPos = searchPos+1;
 	}
 	
-	if (searchPrevPos < str.length()){
+	if (searchPrevPos <= str.length())
```

Comment:
Why < becomes <=?

Reply:
In the current implementation, if we have a separator at the end of the string like so: 
“Hperigo,vinjn,” 

we have the following result:
```
0. 'Hperigo' 
1. 'vinjn' 
2. '' 
```
So the  '<=' is there just to copy the current behavior. It appends the last character if it's a separator, but I will admit that if we just leave < it the result looks better:

```
0. 'Hperigo' 
1. 'vinjn'
```